README.md

```diff
@@ -39,17 +39,23 @@ First, make sure that you have the following dependencies:
 [Node]: https://nodejs.org/en/
 [Yarn]: https://yarnpkg.com/lang/en/
 [GitHub API token]: https://github.com/settings/tokens
-
 [macos-gnu]: https://github.com/sourcecred/sourcecred/issues/698#issuecomment-417202213
 
+When you create a token, a good strategy is to write it into a local file that
+is ignored by the repository. For example, we can write it into `.token`.
```

Comment:
We’ve generally avoided explicit recommendations as to where to store
tokens. For instance, if storing in a token file, you’d need to be sure
to `chmod go-rwx` the token file. Adding it to the `.gitignore` is
obviously a requirement, but an accidental `git add -f .` will still
immediately copy it into the—unsecured!—Git object store as a new blob.
Tarring up the whole repository directory would include the secret, too.
It’ll be removed by a `git clean -qfdx`. And so on.

Can we remove the `.token` suggestions from the gitignore (etc.) and
copy, and instead only describe that the token needs to be provided as
an environment variable?

Reply:
Sure, works for me.

Reply:
Thank you!